File: sdks/thunkmetrc-py/wrapper/src/thunkmetrc/wrapper/models/batch.py
from typing import TypedDict, List, Optional, Any, Generic, TypeVar, TYPE_CHECKING
from typing_extensions import TypedDict

if TYPE_CHECKING:
    from . import *

class Batch(TypedDict, total=False):
    Id: int
    ItemCategories: List['LabTestsBatchItemCategoriesItem']
    ItemCategoryCount: int
    LabTestTypeCount: int
    LabTestTypes: List['LabTestsBatchLabTestTypesItem']
    Name: str
    RequiresAllFromLabTestBatch: bool

File: sdks/thunkmetrc-ts/wrapper/src/models/Batch.ts

import type { LabTestsBatchItemCategoriesItem } from './LabTestsBatchItemCategoriesItem';
import type { LabTestsBatchLabTestTypesItem } from './LabTestsBatchLabTestTypesItem';
export interface Batch {
    Id?: number;
    ItemCategories?: LabTestsBatchItemCategoriesItem[];
    ItemCategoryCount?: number;
    LabTestTypeCount?: number;
    LabTestTypes?: LabTestsBatchLabTestTypesItem[];
    Name?: string;
    RequiresAllFromLabTestBatch?: boolean;
}
